6.6/Dockerfile

```diff
@@ -76,11 +76,19 @@ RUN set -e; \
   chown -R "$SOLR_USER:$SOLR_GROUP" /opt/solr /docker-entrypoint-initdb.d /opt/docker-solr && \
   chown -R "$SOLR_USER:$SOLR_GROUP" /opt/mysolrhome
 
```

Comment:
Don't edit the individual Dockerfiles. Modify the `Dockerfile*template` files, then run `tools/update.sh` to generate all the Dockerfiles.

Reply:
Done.